nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,89 @@
+const express = require("express");
+const app = express();
+
+// import data here
+const meals = require("./data/meals");
+const reservations = require("./data/reservations");
+const reviews = require("./data/reviews");
+
+//Adding a function to get the reviews for every meal
+function getReviews() {
+  meals.map((meal) => {
+    let matchedReviews = [];
+    reviews.map((review) => {
```

Comment:
you can use a filter instead, you can read more about it here:
https://medium.com/poka-techblog/simplify-your-javascript-use-map-reduce-and-filter-bd02c593cc2d